go/private/actions/compilepkg.bzl

```diff
@@ -82,7 +82,7 @@ def emit_compilepkg(
     if bool(nogo) != bool(out_facts):
         fail("nogo must be specified if and only if out_facts is specified")
 
-    inputs = (sources + embedsrcs + [go.package_list] +
+    inputs = (sources + embedsrcs + [go.sdk.package_list] +
```

Comment:
Only vaguely related: I wonder what we could save if we used depsets here.

Reply:
we can try in a follow-up :)